kpatch-build/create-diff-object.c

```diff
@@ -1536,7 +1564,7 @@ static void kpatch_replace_sections_syms(struct kpatch_elf *kelf)
 				continue;
 			}
 
-#ifdef __powerpc64__
+#if defined(__powerpc64__) || defined(__aarch64__)
```

Comment:
Just curious, why `add_off ` is `0` in aarch64?
I see different handling in x86 below. 
:)